packages/contentful-slatejs-adapter/src/slatejs-to-contentful-adapter.ts

```diff
@@ -47,7 +41,12 @@ function convertNode(
       content: [],
     };
     if (!schema.isVoid(contentfulElement)) {
-      contentfulElement.content = flatMap(node.children, childNode => convertNode(childNode, schema));
+      contentfulElement.content = flatMap(node.children, childNode =>
+        convertNode(childNode, schema),
+      );
+    }
+    if (contentfulElement.content.length === 0 && schema.isTextContainer(node.type)) {
+      contentfulElement.content.push(convertText({ text: '', data: {} }));
```

Comment:
This change isn't strictly necessary right now, but I think it makes things more predictable as otherwise, it'll be a SlateJS implementation detail whether a CF entry's RT field edited with our RT editor includes paragraphs with empty `content: []` nodes or `content` nodes with an empty text node.

SlateJS implementation detail is always bad as on any given upgrade their implementation could change without us noticing and impacting the document structure that customers might expect after editing their entry with e.g. the web app.

Reply:
Actually, just realized this should remove the necessity for `sanitizeSlateDoc()` in the new RTE, although, not sure that could result in some crashes as text leaves might not be in the Slate doc then until the editor is reloaded and the cf-to-slate adapter ensures their presence :thinking: